bundles/org.openhab.binding.dirigera/README.md

```diff
@@ -0,0 +1,781 @@
+| `sunrise`       | DateTime  | R          | Date and time of next sunrise                |          |
+| `sunset`        | DateTime  | R          | Date and time of next sunset                 |          |
+| `statistics`    | String    | R          | Several statistics about gateway activities  |          |
+
+Channel `location` can overwrite GPS position with openHAB location, but it's not possible to delete GPS data.
+See [Gateway Limitations](#gateway-limitations) for further infomration.
+
+### Follow Sun
+
+<img align="right" height="100" src="doc/follow-sun.png">
+
+[Motion Sensors](#motion-sensor) can be active all the time or follow a schedule.
+One schedule is follow the sun which needs to be activated in the DIRIGERA GATEWAY.
+
+## Things
+
+With [DIRIGERA Gateway Bridge](#gateway-bridge) in place things can be connected as mentioned in the [supported things section](#supported-things).
+Things contain generic [configuration](), [properties]() and [channels]() according to their capabilities.
+
+### Generic Thing Configuration
+
+Each thing is identified by a unique id which is mandatory to configure.
+Discovery will automatically identify the id.
+
+| Name              | Type    | Description                         | Default | Required |
+|-------------------|---------|-------------------------------------|---------|----------|
+| `id`              | text    | Unique id of this device / scene    | N/A     | yes      |
+
+### Generic Thing Properties
+
+Each thing has properties attached for identification.
+
+<img align="center" width="500" src="doc/thing-properties.png">
+
+### Generic Thing Channels
+
+#### OTA Channels
+
+Over-the-Air (OTA) updates are common for many devices. 
+If device is providing these channels is detected during runtime.
+
+| Channel         | Type      | Read/Write | Description                                  | Advanced |
+|-----------------|-----------|------------|----------------------------------------------|----------|
+| `ota-status`    | Number    | R          | Over-the-air overall status                  |          |
+| `ota-state`     | Number    | R          | Over-the-air current state                   |    X     |
+| `ota-progress`  | Number    | R          | Over-the-air current progress                |    X     |
+
+`ota-status` shows the _overall status_ if your device is _up to date_ or an _update is available_.
+`ota-state` and `ota-progress` shows more detailed information which you may want to follow, that's why they are declared as advanced channels.
+
+**OTA Mappings**
+
+Mappings for `ota-status`
+
+- 0 : Up to date
+- 1 : Update available
+
+Mappings for `ota-state`
+
+- 0 : Ready to check
+- 1 : Check in progress
+- 2 : Ready to download
+- 3 : Download in progress
+- 4 : Update in progress
+- 5 : Update failed
+- 6 : Ready to update
+- 7 : Check failed
+- 8 : Download failed
+- 9 : Update complete
+- 10 : Battery check failed
```

Comment:
You could provide the channels with option lists in the xml, this would prevent any mapping, make the UI able to provide presets and would make translation easier.

Reply:
[It's already in](https://github.com/openhab/openhab-addons/pull/17719/files#diff-bd10fdfb32ffe023806246406e278b520aed1a4891d95cbe3ec40150748971baR309)
For usage in UI these mappings are not needed.
For rules it's useful to have these mappings to check state updates and sending commands.